src/applications/ezr/containers/IntroductionPage.jsx

```diff
@@ -51,7 +47,7 @@ const IntroductionPage = ({ fetchEnrollmentStatus, route }) => {
             <>
               <ProcessDescription />
               {isUserLOA1 ? (
-                <IdentityVerificationAlert onVerify={onVerifyEvent} />
+                <IdentityVerificationAlert />
```

Comment:
This should be `<VerifyAlert headingLevel={3} dataTestId="ezr-identity-alert" />`

Reply:
Lets also wrap that in a `div` with the class of `vads-u-margin-y--4` to keep the margin that was around the alert.